cmd/options/options.go

```diff
@@ -9,6 +9,25 @@ import (
 	cliflag "k8s.io/component-base/cli/flag"
 )
 
+type TokenPassthroughOptions struct {
+	Audiences []string
+	Enabled   bool
+}
+
+func (t *TokenPassthroughOptions) AddFlags(fs *pflag.FlagSet) {
+	fs.StringSliceVar(&t.Audiences, "token-audiences", t.Audiences, ""+
```

Comment:
Maybe `token-passthrough-audiences` would make it clearer, when this flag is used